SU-SWS/stanford_basic
# D8CORE-1099: Added a z-index to the masthead to keep the shadow on top

# READY FOR REVIEW

# Summary
- Changed the header to have a z-index:1. 
- Look at why the base.js file changed? It is because of Bourbon?

# Review By (Date)
- EOD 1/30 for making the sprint

# Urgency
- Nice to have

# Steps to Test

1. Checkout this branch and clear caches.
2. Verify the header shadow appears on the image and still visible when there is no image. Change is on the header.su-masthead

# Affected Projects or Products
- The Stanford Basic theme. D8 Core product

# Associated Issues and/or People
- D8CORE-1099
- @kerri-augenstein FYI

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
